### PR TITLE
adding files of second lab

### PR DESCRIPTION
**Pull request by d3nnyyy**

### CODE REVIEW

File: AddChair.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package ua.lviv.iot.algo.part1.lab1;
+
```
Comment (Danvace) [inline AddChair.java:2]: remove this line

File: Chair.java [hunk 2]
```diff
@@ -1,63 +1,14 @@
...
-
-}
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+
+@AllArgsConstructor
+@Setter
+@Getter
+@ToString
+
```
Comment (Danvace) [inline Chair.java]: remove this line

File: Chair.java [hunk 3]
```diff
@@ -1,63 +1,14 @@
...
+abstract public class Chair {
+
+    protected int maxWeight;
+    protected String material;
```
Comment (Danvace) [inline Chair.java:11]: this variable should be private

File: Chair.java [hunk 4]
```diff
@@ -1,63 +1,14 @@
...
+    public abstract void adjustPosition(int value);
+}
```
Comment (Danvace) [inline Chair.java:13]: add the last line

File: ChairManager.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.Getter;
+import lombok.ToString;
+
```
Comment (Danvace) [inline ChairManager.java]: remove this line

File: ChairManager.java [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+
```
Comment (Danvace) [inline ChairManager.java]: remove this line

File: ChairManager.java [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+        Chair reclinerChair2 = new ReclinerChair(180, "Fabric", 100);
+
+        chairManager.addChair(feedingTable1);
+        chairManager.addChair(feedingTable2);
+        chairManager.addChair(officeChair1);
+        chairManager.addChair(officeChair2);
+        chairManager.addChair(gamingChair1);
+        chairManager.addChair(gamingChair2);
+        chairManager.addChair(reclinerChair1);
+
```
Comment (Danvace) [inline ChairManager.java:58]: remove this line

File: ChairManager.java [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        chairManager.addChair(reclinerChair2);
+//
+//        for (Chair chair : chairs){
+//            System.out.println(chair);
+//        }
```
Comment (Danvace) [inline ChairManager.java]: do not push commented code on github

File: ChairManager.java [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+        List<Chair> chairsWithCertainMaterial = chairManager.findChairsByMaterial("Leather");
+        chairsWithCertainMaterial.stream().forEach(s -> System.out.println(s));
+
+        List<Chair> chairsWithCertainMaxWeight = chairManager.findChairsWithMaxWeightMoreThan(120);
+        chairsWithCertainMaxWeight.stream().forEach(s -> System.out.println(s));
+
+    }
+}
```
Comment (Danvace) [inline ChairManager.java:72]: add last line

File: FeedingTable.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.Getter;
+
+@Getter
+
```
Comment (Danvace) [inline FeedingTable.java]: remove this line

File: FeedingTable.java [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+public class FeedingTable extends Chair {
+    private static double minHeight = 0.5;
```
Comment (Danvace) [inline FeedingTable.java]: add empty line

File: GamingChair.java [hunk 12]
```diff
@@ -0,0 +1,37 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.Getter;
+
+@Getter
+
```
Comment (Danvace) [inline GamingChair.java]: remove this line

File: GamingChair.java [hunk 13]
```diff
@@ -0,0 +1,37 @@
+package ua.lviv.iot.algo.part1.lab1;
+
```
Comment (Danvace) [inline GamingChair.java:2]: remove this line

File: GamingChair.java [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+import lombok.Getter;
+
+@Getter
+
+public class GamingChair extends Chair {
+    private static double minHeight = 1.2;
```
Comment (Danvace) [inline GamingChair.java]: add empty line

File: OfficeChair.java [hunk 15]
```diff
@@ -0,0 +1,31 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+public class OfficeChair extends Chair{
```
Comment (Danvace) [inline OfficeChair.java:3]: add empty line

File: OfficeChair.java [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+        } else {
+            this.angle = minAngle;
+        }
+    }
+
+    @Override
+    public String toString(){
+        return "OfficeChair has an angle of " + angle + "degrees";
+    }
+}
```
Comment (Danvace) [inline OfficeChair.java:32]: add last empty line

File: ReclinerChair.java [hunk 17]
```diff
@@ -0,0 +1,32 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+public class ReclinerChair extends Chair {
```
Comment (Danvace) [inline ReclinerChair.java:3]: add empty line

File: AddChair.java [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+import java.util.List;
+
+public abstract class AddChair {
+
+    abstract List<Chair> findChairsByMaterial(String material);
+
+    abstract List<Chair> findChairsWithMaxWeightMoreThan(int maxWeight);
+
+    abstract void addChair(final Chair chair);
+}
```
Comment (Danvace) [inline AddChair.java:12]: add last empty line

**Review by d3nnyyy — COMMENTED** [review]
fixed

**Review by Danvace — APPROVED** [review]

**Pull request merged**